examples/step-7/step-7.cc

```diff
@@ -868,15 +868,30 @@ namespace Step7
                                         VectorTools::H1_seminorm);
 
     // Finally, we compute the maximum norm. Of course, we can't actually
-    // compute the true maximum, but only the maximum at the quadrature
-    // points. Since this depends quite sensitively on the quadrature rule
-    // being used, and since we would like to avoid false results due to
-    // super-convergence effects at some points, we use a special quadrature
-    // rule that is obtained by iterating the trapezoidal rule by the degree of
-    // the finite element times two plus one in each space direction.
-    // Note that the constructor of the QIterated class
-    // takes a one-dimensional quadrature rule and a number that tells it how
-    // often it shall use this rule in each space direction.
+    // compute the true maximum of the error over *all* points in the domain,
+    // but only the maximum over a finite set of evaluation points that, for
+    // convenience, we will still call "quadrature points" and represent by
+    // an object of type Quadrature) even though we do not actually perform any
```

Comment:
```suggestion
    // an object of type Quadrature even though we do not actually perform any
```
I couldn't find the counterpart.